scheduler/scheduler-smartproxy-common/src/main/java/org/ow2/proactive/scheduler/smartproxy/common/AbstractSmartProxy.java

```diff
@@ -863,6 +902,9 @@ protected void updateTask(NotificationData<TaskInfo> notification) {
                 jobTracker.removeAwaitedTask(id.toString(), tname);
                 break;
             }
+            default: {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Extract this nested code block into a method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1199)